zap/src/main/dist/zap.sh

```diff
@@ -67,11 +67,16 @@ if [ -f "$JVMPROPS" ]; then
 elif [ "$OS" = "Linux" ]; then
     HOSTMEM=$(expr $(sed -n 's/MemTotal:[ ]\{1,\}\([0-9]\{1,\}\) kB/\1/p' /proc/meminfo) / 1024)
     if [ "$IS_CONTAINERIZED" = "true" ]; then
-        # If we are running in a container, we need to use the cgroup memory limit
-        MEMLIMIT=$(grep hierarchical_memory_limit /sys/fs/cgroup/memory/memory.stat | cut -d' ' -f2)
-        MEM=$(expr $MEMLIMIT / 1024 / 1024)
-        # If the cgroup memory limit is not set, use the host memory
-        if [ "$MEMLIMIT" -eq "9223372036854771712" ]; then
+        # Check if memory.stat exists to determine if we are running in a container 
```

Comment:
This comment is not correct, that's the previous check.

Reply:
You're right, I would suggest something like "Check if memory.stat exists so it can be used to determine the memory limit" ?

Reply:
Looks good.